shapes/contact/group.shex

```diff
@@ -0,0 +1,26 @@
+PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
+PREFIX xsd:   <http://www.w3.org/2001/XMLSchema#>
+
+# This shape describes some of the information we store when adding a group to
+# an address book. The resource it is used for is to describe the group itself
+# (meaning the resource should only contain one group). We use the id "#this"
+# to refer to the subject of the group.
+
+# This model was originally developed as part of the
+# SolidOS/Data Browser project, and we strive to keep it interoperable with
+# their work. Some custom terms have been introduced to vocabularies we think
+# they could belong to (e.g. vcard:includesGroup). We'll make a note of the
+# custom term when used.
+
+vcard:Group {
+  vcard:fn xsd:string {1} ;
+  vcard:note xsd:string ? ;
+  vcard:hasMember vcard:Individual * OR vcard:Organization * ;
```

Comment:
I'm not familiar with this syntax, does it mean that you can have a mix of 0 to many Individuals and Organizations or 0 to many of one or the other?

Reply:
The latter - the '*' just means 'zero of more', so this is saying the `vcard:hasMember` predicate has the range zero or more `vcard:Individual`, or zero or more `vcard:Organization`.